app/components/global/common/CountDown.js

```diff
@@ -2,8 +2,8 @@ import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import worldmap from "./worldmap.svg";
 
-const challengeStartDate = new Date(2021, 6, 1, 0, 0, 0, 0); // 8th November 
-const challengeEndDate = new Date(2021, 9, 1, 0, 0, 0, 0); // 22nd November Midnight
+const challengeStartDate = new Date(2023, 4, 26, 0, 0, 0, 0); // 8th November 
```

Comment:
those dates are different from the db record. Wouldn't that cause a havoc?

Plus these date comment